tests/test_ifelse.py

```diff
@@ -527,6 +520,30 @@ def test_str(self):
         res.owner.op.as_view = True
         assert str(res.owner).startswith("if{name,inplace}")
 
+    def test_static_branch_shapes(self):
```

Comment:
Can we check instead a case where one branch is shape=(1,) and the other shape=(2,)? Just because some Type error checks might still be lurking around in the codebase.